### ISSUE TITLE
Arithmetic helpers

### ISSUE THREAD

**Issue by tomakehurst** [tomakehurst/wiremock#1526]
Support basic mathematical operations via a helper.

Could work like this?

```handlebars
{{m 2 '+' 5}}
{{m 10 '-' 3}} 
{{m 2.5 '*' 4.3}}
{{m 12 '/' 0.5}}
{{m myVal '%' 2}}
```

Probably use this: https://github.com/mrhanlon/handlebars-math-helper/blob/master/src/main/java/edu/utexas/tacc/MathHelper.java

**Comment by tomakehurst** [comment]
Need to look a bit more closely at how this works in terms of type coercion i.e. what happens when params are strings vs. numbers, and should the return type necessarily be a string?

**Comment by tomakehurst** [comment]
Implemented in c40a2850706967bdf2c6d4f578ac34040e32db4e

**Comment by tomakehurst** [comment]
Implemented in c40a2850706967bdf2c6d4f578ac34040e32db4e
